tests/DoctrineTest/InstantiatorPerformance/InstantiatorPerformanceEvent.php

```diff
@@ -13,59 +16,57 @@
  */
 class InstantiatorPerformanceEvent
 {
-    /**
-     * @var \Doctrine\Instantiator\Instantiator
-     */
+    /** @var Instantiator */
     private $instantiator;
 
     public function init() : void
     {
         $this->instantiator = new Instantiator();
 
-        $this->instantiator->instantiate(__CLASS__);
-        $this->instantiator->instantiate('ArrayObject');
-        $this->instantiator->instantiate('DoctrineTest\\InstantiatorTestAsset\\SimpleSerializableAsset');
-        $this->instantiator->instantiate('DoctrineTest\\InstantiatorTestAsset\\SerializableArrayObjectAsset');
-        $this->instantiator->instantiate('DoctrineTest\\InstantiatorTestAsset\\UnCloneableAsset');
+        $this->instantiator->instantiate(self::class);
```

Comment:
Do we have a rule for this changes?
```diff
-__CLASS__
+self::class
```

Reply:
No.

Reply:
So, why? :)

Reply:
Because other places already use `::class`, and _just because_ it's better. :D

Reply:
Tbh, I'd fancy such a rule - it would remove the ambiguity once and for all :+1: